platform/ui/src/components/downloadDialog/DownloadDialog.js

```diff
@@ -125,10 +133,13 @@ const DownloadDialog = ({
     updateViewportPreview,
     fileType,
     downloadCanvas.ref,
+    minimumSize,
+    maximumSize,
   ]);
 
-  const onHeightChange = () => {
-    const newHeight = event.target.value;
+  const onHeightChange = event => {
+    const newHeight =
```

Comment:
You can also use Math.min(event.target.value, maximumSize) if you want to easily clamp something.

Reply:
That's a lot better, thx.